src/components/Events/EventsBookingForm.jsx

```diff
@@ -48,6 +48,122 @@ const EventsBookingForm = ({ slug, price, product }) => {
     },
   });
 
+  const handleSendEmail = async (data) => {
```

Comment:
Why is this part in the front end? What happens if the `handleSubmitBookingRequest` `mutateAsync` has an error? Also, the admins' email and name should not be on the client side; that's business data that does not belong here.

Reply:
@Harithmetic1 

If the handleSubmitFails. Then send email should Ideally fail as well. I could move this to the backend it would probably be easier.